src/nvim/api/vim.c

```diff
@@ -703,6 +703,24 @@ ArrayOf(String) nvim_list_runtime_paths(void)
   return rv;
 }
 
+ArrayOf(String) nvim_find_runtime_file(String name, Boolean all)
```

Comment:
maybe not worth adding a new "verb", I think "get" works here.

```suggestion
ArrayOf(String) nvim_get_runtime_file(String name, Boolean all)
```

Reply:
yea that should work